libexec/rc/rc.d/motd

```diff
@@ -53,6 +53,11 @@ append_perf_warnings()
 		echo -n " CHERI_CAPREVOKE"
 		echo "WARNING: capability revocation enabled by default, this may affect performance" >> $_t
 	fi
+	runtime_revocation_every_free_default=`sysctl -n security.cheri.runtime_revocation_every_free_default 2> /dev/null`
+	if [ "${runtime_revocation_every_free_default}" = "1" ]; then
+		echo -n " REVOKE_EVERY_FREE"
+		echo "WARNING: capability revocation enabled on every free by default, this debugging feature will greatly affect performance" >> $_t
```

Comment:
I might say `expect greatly reduced performance`, more in line with the WITNESS and INVARIANTS warnings.

Reply:
"WARNING: capability revocation on every free debugging feature enabled by default, expect greatly reduced performance" then?

Reply:
(bordering on word salad but the best phrasing I can think of)

Reply:
Sounds good enough :)